src/app/containers/App/App.jsx

```diff
@@ -64,7 +66,7 @@ export const App = ({ routes, location, initialData, bbcOrigin }) => {
       };
       fetchData();
     }
-  }, [routes, location.pathname]);
+  }, [routes, location.pathname, dials]);
```

Comment:
If the values for `dials` change this would trigger the effect and re-render the page. Are the dials going to change on a client side render? if not I'd expect this could be removed.

Reply:
The value (and contents) of `dials` should never change, I added this simply because without it you get a React warning about an undeclared dependency for `useEffect` but I'm hazy about hooks so if you know of a better solution I'm all ears :-)

Reply:
Hmmm I'm not seeing this error, was it a build time or a linting error?

Reply:
Linting. If I remove it I get:
```
React Hook useEffect has a missing dependency: 'dials'. Either include it or remove the dependency array. If 'setState' needs the current value of 'dials', you can also switch to useReducer instead of useState and read 'dials' in the reducer  react-hooks/exhaustive-deps
```

Reply:
hmm ok I do not 100% understand what this linting rule is doing. However there is no harm having this here, especially as the dials value wont be updating anyway.

happy to leave this here too.

Reply:
I've put it back in on the basis that we do not expect `dials` to ever change after initial server render, but the other options (like disabling the eslint rule with a comment) all create other risks like someone missing an important warning later on. Thanks for querying.